scripts/delete_preflight_apps.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -eo pipefail
+if [ "${FLY_PREFLIGHT_TEST_FLY_ORG}x" = "x" ] ; then
```

Comment:
`if [ "${FLY_PREFLIGHT_TEST_FLY_ORG}" == "" ] ; then` should work here.

https://www.shellcheck.net/wiki/SC2268

Reply:
np, will change. please use ["insert a suggestion"](https://haacked.com/archive/2019/06/03/suggested-changes/) for nits like this in the future, so the pr author can do the 1 click accept to incorporate the change. thx!